src/MqttController.js

```diff
@@ -30,31 +30,37 @@ client.on('message', function (topic, message) {
   }}
 })
 
-export function forward(){
-  client.publish('/smartcar/control/throttle', '50')
+export function forward(speed){
+  client.publish('/smartcar/control/throttle',"'"+speed+"'")
 }
 
-export function backward(){
-  client.publish('/smartcar/control/throttle', '-50')
+export function backward(speed){
+  client.publish('/smartcar/control/throttle',"'"+speed+"'")
 }
 
-export function left(){
-  client.publish('/smartcar/control/steering', '-30')
+export function left(speed){
+  client.publish('/smartcar/control/steering',"'"+speed+"'")
 }
 
-export function right(){
-  client.publish('/smartcar/control/steering', '30')
+export function right(speed){
+  client.publish('/smartcar/control/steering',"'"+speed+"'")
 }
 
-export function stopTurn(){
-  client.publish('/smartcar/control/steering', '0')
+export function stopTurn(speed){
+  client.publish('/smartcar/control/steering',"'"+speed+"'")
 }
 
-export function stopSpeed(){
-  client.publish('/smartcar/control/throttle', '0')
+export function stopSpeed(speed){
+  client.publish('/smartcar/control/throttle',"'"+speed+"'")
 }
```

Comment:
This function doesent make sense to me shouldnt it default to 0 speed instead of taking speed as an argument as the function is called stopSpeed?

Reply:
and perhaps the left and right functions aren't needed now that everything takes input and they could rely on movex and movey instead